workers/unity/Packages/io.improbable.gdk.buildsystem/Configuration/BuildContext.cs

```diff
@@ -12,10 +12,11 @@ public struct BuildContext
         public BuildTargetConfig BuildTargetConfig;
         public BuildEnvironment BuildEnvironment;
         public ScriptingImplementation ScriptingImplementation;
+        public iOSSdkVersion? IosSdkVersion;
 
         public static List<BuildContext> GetBuildContexts(IEnumerable<string> wantedWorkerTypes,
             BuildEnvironment buildEnvironment, ScriptingImplementation? scriptImplementation = null,
-            ICollection<BuildTarget> buildTargetFilter = null)
+            ICollection<BuildTarget> buildTargetFilter = null, iOSSdkVersion? iosSdkVersion = null)
```

Comment:
This method signature is starting to get chonky

Reply:
Agreed, might need a builder pattern if this needs to grow further

Reply:
Or wrap all the optional args into a struct